llvm/lib/Analysis/ReplayInlineAdvisor.cpp

```diff
@@ -43,8 +43,8 @@ ReplayInlineAdvisor::ReplayInlineAdvisor(
   //   main:3:1.1;
   // We use the callsite string after `at callsite` to replay inlining.
   line_iterator LineIt(*BufferOrErr.get(), /*SkipBlanks=*/true);
-  const std::string PositiveRemark = "' inlined into '";
-  const std::string NegativeRemark = "' will not be inlined into '";
+  static const std::string PositiveRemark = "' inlined into '";
```

Comment:
StringRef PositiveRemark?

Reply:
I don't understand the question.

Reply:
You can use string ref here
```
StringRef PositiveRemark = "' inlined into '";
```

However I don't understand why you can report in code as is